### ISSUE TITLE
Skip to new member button from lookup page

### ISSUE THREAD

**Issue by ssbarsky** [jordanbrown0/reg3#463]
Once the clerk determines that the person is not already in the database, it would be handy for there to be a button (hot key) to jump directly to "New Member".

**Comment by jordanbrown0** [comment]
Yes... I keep thinking about some integration between the two that encourages you to try looking somebody up before you add them.

The biggest problem is that the typing spent looking them up (and failing) is likely wasted; especially with the free-form search kind of lookup there's no obvious way to take that input and start to populate the form.

But indeed maybe the right integration is for there to be a single "Member" entry on the main menu, that leads to a lookup interface with a "new" button.  There would definitely need to be a hot key; I don't want people to have to move to the mouse for anything in the routine workflows.  (They do anyway, and it's frustrating to watch.)

**Comment by ssbarsky** [comment]
Yes, since the search is so broad (people may check street address or email to triple check that the person isn't there) that it is sad not to use that data. Dave C mentioned that it was hard training people to check first before entering new. So forcing the lookup BEFORE entering new may be a good thing.
